src/v2/Apps/Artist/Routes/AuctionResults/ArtistAuctionResultItem.tsx

```diff
@@ -542,12 +544,16 @@ const renderLargeCollapse = (props, user, mediator, filtersAtDefault) => {
         <Row>
           <Col sm={2}>
             <Text variant="xs" fontWeight="bold">
-              Artwork Info
+              {categoryText !== "Unknown"
+                ? "Artwork Info"
+                : "Artwork Dimension"}
```

Comment:
Added the condition for the title as well. As if `categoryText` is unknown we will have only dimension_text

Reply:
I think this should be `dimensions` instead of `dimension`